plaidml/op/op_test.cc

```diff
@@ -32,48 +33,42 @@ Program makeProgram(const std::string& name, const std::vector<Tensor>& outputs)
   return ProgramBuilder(name, outputs).compile();
 }
 
-TEST(Op, Abs) {
-  auto I = Placeholder(DType::FLOAT32, {1, 224, 224, 3}, "I");
-  auto abs = op::abs(I);
-  auto program = makeProgram("abs", {abs});
-  IVLOG(1, program);
+TEST_F(OpTest, Abs) {
+  auto A = Placeholder(DType::FLOAT32, {10, 20});
+  auto program = makeProgram("abs", {op::abs(A)});
+  std::cout << program << std::endl;
 
-  EXPECT_THAT(program, Eq(R"#(
+  // clang-format off
+  // CHECK-LABEL: OpTest.Abs
+  // CHECK: func @abs
+  // CHECK: %[[cst:.*]] = "eltwise.sconst"() {value = 0.000000e+00 : f64} : () -> tensor<f32>
+  // CHECK: %{{.*}} = "eltwise.neg"(%{{.*}}) : (tensor<10x20xf32>) -> tensor<10x20xf32>
+  // CHECK: %{{.*}} = "eltwise.cmp_lt"(%{{.*}}, %[[cst]]) : (tensor<10x20xf32>, tensor<f32>) -> tensor<10x20xi1>
+  // CHECK: %{{.*}} = "eltwise.select"(%{{.*}}, %{{.*}}, %{{.*}}) : (tensor<10x20xi1>, tensor<10x20xf32>, tensor<10x20xf32>) -> tensor<10x20xf32>
+  // CHECK: return %{{.*}} : tensor<10x20xf32>
+  // clang-format on
 
-module {
-  func @abs(%arg0: tensor<1x224x224x3xf32> {tile.name = "I"}) -> tensor<1x224x224x3xf32> {
-    %cst = "eltwise.sconst"() {value = 0.000000e+00 : f64} : () -> f32
-    %0 = "eltwise.neg"(%arg0) : (tensor<1x224x224x3xf32>) -> tensor<1x224x224x3xf32>
-    %1 = "eltwise.cmp_lt"(%arg0, %cst) : (tensor<1x224x224x3xf32>, f32) -> tensor<1x224x224x3xi1>
-    %2 = "eltwise.select"(%1, %0, %arg0) : (tensor<1x224x224x3xi1>, tensor<1x224x224x3xf32>, tensor<1x224x224x3xf32>) -> tensor<1x224x224x3xf32>
-    return %2 : tensor<1x224x224x3xf32>
-  }
-}
-)#"));
+  runProgram(program);
 }
 
-TEST(Op, All) {
+TEST_F(OpTest, All) {
   auto I = Placeholder(DType::FLOAT32, {1, 224, 224, 3}, "I");
   auto program = makeProgram("all", {op::all(I)});
-  IVLOG(1, program);
-  EXPECT_THAT(program, Eq(R"#(
-#map0 = affine_map<() -> ()>
-#map1 = affine_map<(d0, d1, d2, d3) -> (d0, d1, d2, d3)>
-
+  std::cout << program << std::endl;
+
+  // clang-format off
+  // CHECK-LABEL: OpTest.All
+  // CHECK: func @all
+  // CHECK: %[[cst1:.*]] = "eltwise.sconst"() {value = 1 : i64} : () -> tensor<si32>
+  // CHECK: %[[cst0:.*]] = "eltwise.sconst"() {value = 0 : i64} : () -> tensor<si32>
+  // CHECK: %{{.*}} = "eltwise.cmp_eq"(%{{.*}}, %[[cst0]]) : (tensor<1x224x224x3xf32>, tensor<si32>) -> tensor<1x224x224x3xi1>
+  // CHECK: %{{.*}} = "eltwise.select"(%{{.*}}, %[[cst0]], %[[cst1]]) : (tensor<1x224x224x3xi1>, tensor<si32>, tensor<si32>) -> tensor<1x224x224x3xsi32>
+  // CHECK: %{{.*}} = tile.contract mul, none, %[[cst1]], %{{.*}} {sink = #map0, srcs = [#map1]} : tensor<si32>, tensor<1x224x224x3xsi32> -> tensor<si32>
```

Comment:
Don't check for specific map names, which could easily change; just `sink = #{{.*}}, srcs = [#{{.*}}]`. You can name them like you do with `cst0` if you think it's important (but I don't think it is -- certainly it's not in this case where you don't include the definitions of the maps or reuse them).